test.s

```diff
@@ -1,3 +1,9 @@
+# to test asm->binary
```

Comment:
This potentially can be wrapped into a sing shell script, that will generate test.s, run gcc and objdump and will ensure that output is what we expect. We can add more tests for each command in the future.